app/models/cdo/edit/exemption-details.js

```diff
@@ -0,0 +1,350 @@
+        type: 'date',
+        id: 'certificateIssued',
+        namePrefix: 'certificateIssued',
+        fieldset: {
+          legend: {
+            text: 'Certificate issued',
+            classes: 'govuk-!-font-weight-bold govuk-!-font-size-16'
+          }
+        },
+        items: [
+          {
+            name: 'day',
+            classes: 'govuk-input--width-2',
+            value: exemption['certificateIssued-day'],
+            attributes: { maxlength: '2' }
+          },
+          {
+            name: 'month',
+            classes: 'govuk-input--width-2',
+            value: exemption['certificateIssued-month'],
+            attributes: { maxlength: '2' }
+          },
+          {
+            name: 'year',
+            classes: 'govuk-input--width-4',
+            value: exemption['certificateIssued-year'],
+            attributes: { maxlength: '4' }
+          }
+        ]
+      },
+      cdoIssued: {
+        type: 'date',
+        id: 'cdoIssued',
+        namePrefix: 'cdoIssued',
+        fieldset: {
+          legend: {
+            text: 'CDO issued',
+            classes: 'govuk-!-font-weight-bold govuk-!-font-size-16'
+          }
+        },
+        items: [
+          {
+            name: 'day',
+            classes: 'govuk-input--width-2',
+            value: exemption['cdoIssued-day'],
+            attributes: { maxlength: '2' }
+          },
+          {
+            name: 'month',
+            classes: 'govuk-input--width-2',
+            value: exemption['cdoIssued-month'],
+            attributes: { maxlength: '2' }
+          },
+          {
+            name: 'year',
+            classes: 'govuk-input--width-4',
+            value: exemption['cdoIssued-year'],
+            attributes: { maxlength: '4' }
+          }
+        ]
+      },
+      cdoExpiry: {
+        type: 'date',
+        id: 'cdoExpiry',
+        namePrefix: 'cdoExpiry',
+        fieldset: {
+          legend: {
+            text: 'CDO expiry',
+            classes: 'govuk-!-font-weight-bold govuk-!-font-size-16'
+          }
+        },
+        items: [
+          {
+            name: 'day',
+            classes: 'govuk-input--width-2',
+            value: exemption['cdoExpiry-day'],
+            attributes: { maxlength: '2' }
+          },
+          {
+            name: 'month',
+            classes: 'govuk-input--width-2',
+            value: exemption['cdoExpiry-month'],
+            attributes: { maxlength: '2' }
+          },
+          {
+            name: 'year',
+            classes: 'govuk-input--width-4',
+            value: exemption['cdoExpiry-year'],
+            attributes: { maxlength: '4' }
+          }
+        ]
+      },
+      court: {
+        label: {
+          text: 'Court',
+          classes: 'govuk-!-font-weight-bold govuk-!-font-size-16'
+        },
+        id: 'court',
+        name: 'court',
+        value: exemption.court,
+        items: [{ text: '', value: null }].concat(courts.map(court => ({
```

Comment:
The prototype has 'Choose court' as the non-selected option. Do you want to add that text?